app.js

```diff
@@ -901,7 +901,7 @@ io.sockets.on('connection',function(socket){
                 openhab.status = 'online';
                 openhab.last_online = new Date;
                 openhab.openhabVersion = socket.handshake.openhabVersion;
-                openhab.myohVersion = socket.handshake.myohVersion;
+                openhab.clientVersion = socket.handshake.clientVersion;
```

Comment:
If you change this here, you also need to update the openhab model so it's persisted to the mongo database, also this is a schema change, so we will need to clean out the old entry from records.